src/crimson/osd/osd_operations/osdop_params.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "messages/MOSDOp.h"
```

Comment:
no need to include `messages/MOSDOp.h` if we don't need the complete type definition of it.